stores/globus.js

```diff
@@ -0,0 +1,118 @@
+const axios = require("axios");
+const utils = require("./utils");
+
+const units = {
+    bli: { unit: "blistr", factor: 1 }, // 1x
+    list: { unit: "sada", factor: 1 }, // 5x
+    pg: { unit: "balení", factor: 1 },
+};
+
+exports.getCanonical = function (item, today) {
+    return utils.convertUnit(
+        {
+            id: "" + item.ean[0],
+            name: item.name,
+            description: item.description,
+            price: item.productInHouse.actualPrice,
+            priceHistory: [{ date: today, price: item.productInHouse.actualPrice }],
+            unit: item.unitId,
+            quantity: item.baseVariantQuantity,
+            url: `${item.name
+                .toLowerCase()
+                .replace(/\s+/g, "-")
+                .match(/[a-z0-9-]+/gi)
+                .join("")}/${item.ean[0]}`,
+            // https://shop.iglobus.cz/cz/mango-zral-balen-rte-2-ks/8594023949733 .../kokosov-oech-1-ks/107200
+            categoryNames: item.productInHouse.placements?.category || item.productCategories[0],
+            ...(item.name && /^Bio[ -]/.test(item.name) && { bio: true }),
+        },
+        units,
+        "globus"
+    );
+};
+
+exports.fetchData = async function () {
+    const categories = {
+        1: "Čerstvé potraviny",
+        2: "Trvanlivé potraviny",
+        4: "Pečivo",
+        5: "Maso, uzeniny",
+        10: "Ostatní",
+        12: "Dům, zahrada",
+        14: "Alkoholické nápoje",
+        15: "Nealkoholické nápoje",
+        16: "Lahůdky",
+        17: "Mražené",
+        18: "Drogerie",
+        19: "Domácnost",
+        20: "Zábava a volný čas",
+        21: "Elektro",
+        22: "Móda",
+        23: "Vlastní výroba",
+        24: "Zdravý svět",
```

Comment:
Tyhle už myslím ani neplatí - tuším, že pak to zas chtělo textové typu cls_czr_wooden_toys, asi 700, ale server jim začal stávkovat, tak jsem to vzal z jiného konce...
Navíc web maj celý divný - třeba kategorie Elektro je prázdná (jestli to mezi tím neopravili), ale když víš, jak ji otevřít, položky existují ??